core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -610,6 +610,22 @@ public enum CoreError implements ScalarDbError {
       "Invalid file extension: %s. Allowed extensions are: %s",
       "",
       ""),
+  DATA_LOADER_INVALID_COLUMN_NON_EXISTENT(
+      Category.USER_ERROR,
+      "0136",
+      "Invalid key: Column %s does not exist in the table %s in namespace %s.",
+      "",
+      ""),
+  DATA_LOADER_INVALID_VALUE_KEY_PARSING_FAILED(
```

Comment:
This doesn't seem to be used?

Reply:
My bad. It has become unused due to refactoring the error handling based on Toshi's feedback. I have removed it now.